verify-docs-format.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+no_entry=false
+no_entry_counter=0
+no_title=false
+no_title_counter=0
+
+# Verify all docs/.../*.md files 
+for file in `find docs -name "*.md" -type f`; do 
+  # 1. TOC check:
+  #    Check they are referenced in at least one of _data/*.yml files.
+  #    Skip checking files in skip_toc_check.txt
+  if ! grep -q "${file}" skip_toc_check.txt; then
+    path=${file%.*}
+    # abc/index.md should point to abc, not abc/index
+    path=${path%%index}
+    if ! grep -q "${path}" _data/*.yml; then
+      echo "Error: ${file} doesn't have an entry in the table of contents under _data/*.yml" 
+      no_entry=true
+      no_entry_counter=$[no_entry_counter+1]
+    fi
+  fi
+
+  # 2. Title check:
+  #    Check they have a proper title.
+  #    Skip checking files in skip_title_check.txt
+  #    Title should start with "title:" and can have several spaces/tabs between
+  #    non-space/tab content. 
+  #    For example, "title:", " title: abc", and "title:" aren't valid, 
+  #    but "title: abc", "title:def" and "title:    def ghi" are both valid.
+  if ! grep -q "${file}" skip_title_check.txt; then
```

Comment:
WDYT of adding everything to skip title check to stop the bleeding, and remove them incrementally as we fix them?